src/pages/student/learnings/LearningList/index.js

```diff
@@ -1,95 +1,101 @@
 import React, { useState, useEffect } from 'react';
-import Card from 'react-bootstrap/Card';
-import { Col, Table } from 'react-bootstrap';
 import { useHistory } from 'react-router-dom';
-
+import { useToast } from '../../../../hooks/useToast';
+import Card from 'react-bootstrap/Card';
 import Pagination from '../../../../components/Pagination';
-import style from './index.module.scss';
+import DataTable from '../../../../components/DataTable';
 import LearningApi from '../../../../api/Learning';
-import CategoryLearned from './component';
+import style from './index.module.scss';
 
 const LearningList = () => {
-  const [learnings, setLearnings] = useState(null);
   const queryParams = new URLSearchParams(window.location.search);
   const pageNum = queryParams.get('page');
+  const sortBy = queryParams.get('sortBy') || '';
+  const sortDirection = queryParams.get('sortDirection') || '';
+  const history = useHistory();
+  const toast = useToast();
 
-  const history = useHistory  ();
-
+  const [learnings, setLearnings] = useState(null);
   const [page, setPage] = useState(pageNum ? parseInt(pageNum) : 1);
   const [perPage, setPerPage] = useState(0);
   const [totalItems, setTotalItems] = useState(0);
   const [lastPage, setLastPage] = useState(0);
+  const [sortOptions, setSortOptions] = useState({
+    sortBy,
+    sortDirection
+  });
```

Comment:
please group together the variables. you can group them by the ones using useState and the ones not using useState